app.js

```diff
@@ -34,26 +31,42 @@ function lockCards(cardsToLock) {
 		card.removeEventListener('click', handleUserInput);
 	});
 	firstCard = null;
+	pausePlay = false;
+
+
 	console.log('someone called lockCards,Cards match');
+
 };
 
 function handleUserInput() {
-	this.classList.add('turned'); // you always want to do this.
+	if (pausePlay) return;
+	this.classList.add('turned');
+	if (this === firstCard) return
 	if (!firstCard) {
-		firstCard = this; //if the firstCard didn't exist, set the first card
+		firstCard = this;
 	} else {
-		currentCard = this; //now it's time to compare both cards!
+		currentCard = this;
+
 		isEqual(firstCard, currentCard);
+
 		isTheSameCard ? lockCards([firstCard, currentCard]) : resetCards([firstCard, currentCard]);
+
 	}
 }
 
 function resetCards(cardsToReset) {
+	pausePlay = true;
 	cardsToReset.forEach((card) => {
 		setTimeout(() => {
-			card.classList.remove('turned')
+			pausePlay = false;
+			card.classList.remove('turned');
+
 		}, 1000);
 	});
 	firstCard = null;
 	console.log('someone called resetCards, Cards do not match');
 }
+
+cards.forEach((card) => {
+	card.addEventListener('click', handleUserInput);
+});
```

Comment:
This would be nicer if you would have put it inside an `init` or `setup` function.

Reply:
Would I have to add:
const cards = document.querySelectorAll('.card');
inside too?